Squirrel/Program.cs

```diff
@@ -16,7 +17,15 @@
 
 services.AddTransient<IEmailSender, EmailSender>();
 
-services.AddAuthentication().AddCookie(CookieAuthenticationDefaults.AuthenticationScheme);
+services.AddAuthentication()
+    .AddCookie(CookieAuthenticationDefaults.AuthenticationScheme)
+    .AddGoogle(GoogleDefaults.AuthenticationScheme, options =>
+    {
+        options.ClientId = configuration["Authentication:Google:ClientId"];
```

Comment:
Some useful tips for possible improvements:
[Options pattern in asp net](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-6.0)
Carry string literals to a static class with constants